docs/source/admin/log-messages.md

```diff
@@ -1,22 +1,20 @@
 # Interpreting common log messages
 
-When debugging errors and outages, looking at the logs emitted by
-JupyterHub is very helpful. This document tries to document some common
-log messages, and what they mean.
+In this subsection of the documentation, you will get to understand the meaning of common log messages and how to resolve them. When debugging errors and outages, it is very helpful to look at the logs emitted by JupyterHub.
```

Comment:
I think the earlier order was preferable: problem statement ("when debugging...") before solution ("explaining common log messages")

Reply:
Thanks for the correction mnirk. I have re-written it.